### PR TITLE
42 UI update

### PR DESCRIPTION
**Pull request by artem-kim**

### CODE REVIEW

**Review by michowski — COMMENTED** [review]
Hi Artem. It would be awesome to add a few tests for this perhaps, as I already have some ideas how we could refactor it to utilize the command names from command classes.

Best!
Arek

**Comment by michowski** [comment]
Also, some conflicts to resolve.

File: src/main/java/newbank/server/NewBankClientHandler.java [hunk 1]
```diff
@@ -66,4 +71,26 @@ private CustomerID logIn() throws IOException {
...
+
+  private void mainMenu() {
+    clearConsole();
+    out.println("Log In Successful");
+    out.println("You have the following options:");
+    out.println("OPTION 1. SHOWMYACCOUNTS — see your accounts and current balance");
+    out.println("OPTION 2. NEWACCOUNT <Name> - open a new account (COMING SOON)"); 
+    out.println("OPTION 3. MOVE <Amount> <From> <To> - move money from one account to another (COMING SOON)");
+    out.println("OPTION 4. PAY <Person/Company> <Ammount> - pay another person (COMING SOON)");
+    out.println("OPTION 5. QUIT - quit the menu"
```
Comment (ZacTredger) [inline src/main/java/newbank/server/NewBankClientHandler.java:93]: Missing a `);` at the end of this line

File: src/main/java/newbank/server/NewBankClientHandler.java [hunk 2]
```diff
@@ -66,4 +71,26 @@ private CustomerID logIn() throws IOException {
...
+  }
+
+  private void mainMenu() {
+    clearConsole();
+    out.println("Log In Successful");
+    out.println("You have the following options:");
+    out.println("OPTION 1. SHOWMYACCOUNTS — see your accounts and current balance");
+    out.println("OPTION 2. NEWACCOUNT <Name> - open a new account (COMING SOON)"); 
+    out.println("OPTION 3. MOVE <Amount> <From> <To> - move money from one account to another (COMING SOON)");
+    out.println("OPTION 4. PAY <Person/Company> <Ammount> - pay another person (COMING SOON)");
```
Comment (ZacTredger) [inline src/main/java/newbank/server/NewBankClientHandler.java:92]: Typo: "Ammount"

**Review by ZacTredger — CHANGES_REQUESTED** [review]
Agree with Arek: It'll need rebasing onto main, which might be quite tricky, so some tests would be a good idea to catch regressions.
Might also be a good idea to do an interactive rebase if you're up to it and squash some of the commits together.

**Comment by jkonan100** [comment]
Morning @artem-kim ,
Could you please catch up with "main", resolve the conflicts and push again? I can't try it as your code is out of synch with main.
Thanks

Kindest Regards,
Jules

**Comment by michowski** [comment]
Heya @artem-kim! Are you continuing on this branch or shall we close the PR?

**Comment by artem-kim** [comment]
Good to close!

**Pull request closed without merging**